backends/imgui_impl_sdl3.cpp

```diff
@@ -71,7 +71,7 @@ struct ImGui_ImplSDL3_Data
     int             PendingMouseLeaveFrame;
     char*           ClipboardTextData;
     bool            MouseCanUseGlobalState;
-
+    ImVec2          RenderScale() const { ImVec2 res; SDL_GetRenderScale(Renderer, &res.x, &res.y); return res;}
```

Comment:
`Renderer` is able to be null in cases where SDL_Renderer isn't being used (IE: OpenGL, etc), so this needs to be able to handle that.